triple-store/triple-store-test/src/test/java/com/powsybl/triplestore/test/TripleStoreSubqueryLimit1Test.java

```diff
@@ -0,0 +1,60 @@
+package com.powsybl.triplestore.test;
```

Comment:
The copyright is missing:

```suggestion
/**
 * Copyright (c) 2025, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 */
package com.powsybl.triplestore.test;
```